cvat/apps/engine/rules/jobs.rego

```diff
@@ -253,3 +253,18 @@ allow if {
     organizations.has_perm(organizations.WORKER)
     is_task_staff
 }
+
+allow if {
+    input.scope in {utils.VIEW_VALIDATION_LAYOUT, utils.UPDATE_VALIDATION_LAYOUT}
+    utils.has_perm(utils.USER)
+    utils.is_sandbox
+    is_task_staff
+}
+
+allow if {
+    input.scope in {utils.VIEW_VALIDATION_LAYOUT, utils.UPDATE_VALIDATION_LAYOUT}
+    input.auth.organization.id == input.resource.organization.id
+    organizations.has_perm(organizations.SUPERVISOR)
+    utils.has_perm(utils.USER)
+    is_task_staff
+}
```

Comment:
We probably need two rules,

One for is_task_staff in organization (Role is not important)
And another one for maintainer  (is_task_staff is not important)

Reply:
Updated